.github/workflows/packages.yml

```diff
@@ -0,0 +1,246 @@
+name: "Build and test packages"
+
```

Comment:
This is a "re-usable worflow". It allows us to share the package build steps among the "main.yml" and the "tag.yml" workflows.